applications/MeshMovingApplication/custom_utilities/explicit_mesh_moving_utilities.cpp

```diff
@@ -66,7 +66,7 @@ namespace Kratos
 
         // Copy the origin model part elements
         auto &r_elems = rOriginModelPart.Elements();
-        for(auto elem : r_elems){
+        for(auto &elem : r_elems){
```

Comment:
so here you were silently creating new elements?


@msandre could such things be avoided with #2527 ?

Reply:
Actually, I need to create new elements. If I'm not wrong, I was iterating with a copy of the elements (no ampersand). So when creating the copy iterator, the base element copy constructor was called instead of the MeshElement one (don't ask me why).

Reply:
@philbucher I don't think so. Explicit is not used for copy constructors.